html ain't easy

GoLangsam opened the pull request:
systematic correction of glitches re html-tags (e.g. forgotten close, orphan </a>, leftovers etc.)
Note: also changed all `<!---` to `<!--` as e.g. pandoc gets sometimes confused by triple dashes.

go101 commented on articles/container.html:
```diff
@@ -910,7 +911,7 @@ <h3>Append And Delete Container Elements</h3>
 which must be followed by three dots <code>...</code>.
 We can learn how to call variadic functions from the article
-<a href="function.html#variadic-call">function types and values</a> later.
+<a href="function.htm#variadic-call">function types and values</a> later.
```
looks you PR is based on an old revision. Here misss a "l", which I have fixed in a newer revision.

go101 commented on articles/interface.html:
```diff
@@ -517,7 +517,7 @@ <h3>Reflection</h3>
 This article will not explain the functionalities provided by
 <a href="https://golang.org/pkg/reflect/">the <code>reflect</code> standard package</a>.
-Please read <!---a href="reflection.html"-->Reflections In Go<!---/a-->
+Please read <a href="reflection.html">Reflections In Go</a>
```
change "Reflections In Go" to "reflections in Go"?
So that it is consistent with other article titles in content?

go101 commented on articles/panic-and-recover-use-cases.html:
```diff
@@ -1,235 +1,235 @@
-</div>
-
-
+<h1>Some Panic/Recover Use Cases</h1>
```
Looks this file is not changed much. Why github diff shows all contents are changed?
Is the article encoding changed?

GoLangsam commented:
Very good late eve to You

never mind - I just started to play with Your good stuff :-)
And as (prev.) Head-of-Dev it's (been) my job to find what nobody else is looking for ;-)

re 1.
- Yeah - I saw You've also been very busy today :-)
- It must have been matters of minutes, as I pulled before I reviewed and added my changes (but my net is slow here)
- I remember having seen some 'confusion' about this link.
  Just, as I saw You just made the page live, I removed the comments.
- Plse be so kind as to change Yourself as You seem fit - I gave @go101 right to my fork

re 2. 
- please see my remark on 'consitent Capitalization' in prev pull#35

re 3.
- I do not know - my last pull of Yours and my fork are same ...

GN

PS: Where else can I find apk's of these gr8 games?
(I do not use store - no like - they only want my data)

go101 commented:
ok, I will make some modifications on you PR and merge it.
Temporarily, I will keep article titles in content lower cases.
They can be modified together in a later modification. 

btw, I sent apk links to your email.
